index.html

```diff
@@ -1 +1,8 @@
-Annah Nyarangi
\ No newline at end of file
+Annah Nyarangi
+About
+I am a beginner programmer, currently working on JavaScript. I look forward to learning more. Currently working as a nursing assistant and hoping to make a career switch from healthcare to Information Technology.
+Experience
+2015 - Current 
+Nursing Assistant, GI Acute Care
+Connect
+https://github.com/anyarangi/intro-to-programming-2023
```

Comment:
Your GitHub link url should be something like `https://github.com/anyarangi`. This link will take you to your GitHub profile, where you can view your repositories, contributions, and more.